sharding/indexHashedNodesCoordinator.go

```diff
@@ -201,18 +210,18 @@ func (ihgs *indexHashedNodesCoordinator) setNodesPerShards(
 	nodesConfig.eligibleMap = eligible
 	nodesConfig.waitingMap = waiting
 	nodesConfig.publicKeyToValidatorMap = ihgs.createPublicKeyToValidatorMap(eligible, waiting)
+	log.Info("from nodesCoordinator")
 	nodesConfig.shardID = ihgs.computeShardForSelfPublicKey(nodesConfig)
 	nodesConfig.selectors, err = ihgs.createSelectors(nodesConfig)
 	if err != nil {
 		return err
 	}
 
-	shardIDForSelfPublicKey := ihgs.computeShardForSelfPublicKey(nodesConfig)
-	nodesConfig.shardID = shardIDForSelfPublicKey
+	log.Warn("Computed shardId", "shardId", nodesConfig.shardID)
```

Comment:
Could be set to Debug level or removed if it is not really necessary

Reply:
remove